tests/unit/CmsZone.spec.ts

```diff
@@ -188,6 +187,7 @@ describe('CmsZone.vue', (): void => {
       await response;
       await localVue.nextTick();
       await localVue.nextTick();
+      await localVue.nextTick();
```

Comment:
how would you feel about copying in this `settled` helper so we don't need to worry about so many nextTicks? https://github.com/propelinc/freshcard-app/blob/master/tests/unit/helpers/run-loop.ts#L1:L9

Reply:
oooo, cool, will try it out!